tests/test_mgxs_library_nuclides/test_mgxs_library_nuclides.py

```diff
@@ -25,8 +25,12 @@ def _build_inputs(self):
         # Initialize MGXS Library for a few cross section types
         self.mgxs_lib = openmc.mgxs.Library(self._input_set.geometry)
         self.mgxs_lib.by_nuclide = True
-        self.mgxs_lib.mgxs_types = ['transport', 'nu-fission',
-                                    'nu-scatter matrix', 'chi']
+        self.mgxs_lib.mgxs_types = ['total', 'transport', 'nu-transport',
```

Comment:
Also, I would lean towards using all `MGXS_TYPES` in each MGXS test. The reason is that we *do* want to ensure that the functionality tested in each test (*i.e.*, energy condensation, HDF5 data stores, etc.) does indeed work for all of our supported MGXS types. If this results in output that is too large for some of the tests, then we can simply hash the "results_true.dat" as we already do for the distribcell test (though this may be less problematic if we use your new pin cell input set everywhere anyway). Thoughts @nelsonag?